_docs/tasks/basic-access-control.md

```diff
@@ -45,31 +45,56 @@ of the `reviews` service. We would like to cut off access to version `v3` of the
    If you log in as any other user (or logout) you should see red ratings stars with each review,
    indicating that the `ratings` service is being called by the "v3" version of the `reviews` service.
 
-1. Explicitly deny access to version `v3` of the `reviews` service. 
+1. Explicitly deny access to version `v3` of the `reviews` service.
 
-   ```bash
-   istioctl mixer rule create global ratings.default.svc.cluster.local -f samples/apps/bookinfo/mixer-rule-ratings-denial.yaml
+   Before setting up the deny rule, we must create a handler and an instance definition that can be used in the deny rule.
+   Replace `metadata.namespace` with the appropriate value.
+   ```yaml
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: denier
+   metadata:
+     name: handler
+     namespace: default
+   spec:
+     status:
+       code: 7
+       message: Not allowed  # This message is sent back the client
+   ---
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: checknothing
+   metadata:
+     name: denyrequest
+     namespace: default
+   spec:
+   ---
    ```
-
-   This command sets configuration for `subject=ratings.default.svc.cluster.local`. 
-   You can display the current configuration with the following command:
-
+   Save the file as mixer-rule-ratings-denial.yaml and run
+   ```bash
+   kubectl apply -f istioctl mixer-rule-ratings-denial.yaml
    ```
-   istioctl mixer rule get global ratings.default.svc.cluster.local
+   You can expect to see the following output
+   ```bash
+   denier "denyall" created
+   checknothing "denyrequest" created
    ```
 
-   which should produce:
-
+   Now create the following rule using the above method
    ```yaml
-   rules:
-   - aspects:
-     - kind: denials
-     selector: source.labels["app"]=="reviews" && source.labels["version"] == "v3"
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: rule
+   metadata:
+     name: denyreviewsv3
+     namespace: default
+   spec:
+     match: destination.labels["app"] == "ratings" && source.labels["app"]=="reviews" && source.labels["version"] == "v3"
+     actions:
+     - handler: denyall.denier
+       instances: [ denyrequest.checknothing ]
```

Comment:
is the value an array or a string that contains an array?

Reply:
`instances` is an array.

Reply:
This should match the style of the configs in istio/mixer.